pkg/tcp/sender.go

```diff
@@ -62,24 +55,26 @@ const (
 )
 
 // LineSender allows you to insert rows into QuestDB by sending ILP
-// messages.
+// messages over TCP(S).
 //
 // Each sender corresponds to a single TCP connection. A sender
 // should not be called concurrently by multiple goroutines.
 type LineSender struct {
-	address       string
-	tlsMode       tlsMode
-	keyId         string // Erased once auth is done.
-	key           string // Erased once auth is done.
-	bufCap        int
-	fileNameLimit int
-	conn          net.Conn
-	buf           *buffer
-	lastMsgPos    int
-	lastErr       error
-	hasTable      bool
-	hasTags       bool
-	hasFields     bool
+	buffer.Buffer
+
+	address string
+
+	// Authentication-related fields
+	tlsMode tlsMode
+	keyId   string // Erased once auth is done.
+	key     string // Erased once auth is done.
+
+	// Auto-flush fields
+	autoFlush     bool
```

Comment:
I don't think we have auto_flush behavior in any other TCP client, e.g. Java's ILP/TCP sender doesn't have it. ILP/TCP senders are kept as is when it comes to flushing/buffer management.

@jerrinot please correct me if I'm wrong.

Reply:
removed from tcp sender